grammars/asciidoc.cson

```diff
@@ -291,21 +300,47 @@ patterns: [
     'captures':
       '1': 'name': 'markup.list.bullet.asciidoc'
   }
+
+  # Includes inline patterns, see repository section below.
+  {
+    'include': '#inline'
```

Comment:
What do you mean here? Because if I put the `include` in the `repository`, it will be a cyclical reference.